design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -57,3 +59,8 @@ type CheckoutPaymentMethod {
 type CartGiftCard {
     code: String!
 }
+
+type CartError {
```

Comment:
Please explain in the proposal why we should ignore other message types like `success`, `notice`, `warning`

Reply:
In base PR https://github.com/magento/graphql-ce/pull/475 we expected all kind of messages. Looks like we need rename CartError to CartMessages.